is ready

JuliaTiunova opened the pull request:
все працює: адаптив, меню, навіть дужки))

JuliaTiunova committed:
Import:
svg;
images;
fonts;
colors

JuliaTiunova committed:
create html structure

JuliaTiunova committed:
add style to header, menu, part section

JuliaTiunova committed:
add style to 'step' section

JuliaTiunova committed:
add style to 'travel' section

JuliaTiunova committed:
add style to 'what' section

JuliaTiunova committed:
style footer

JuliaTiunova committed:
add style to all

JuliaTiunova committed:
adapt to md + screen sizes

JuliaTiunova committed:
add: js, burger, mobile-menu

JuliaTiunova committed:
add a working mobile-menu, adapt to sm size screen

JuliaTiunova committed:
fix input width for mobile

JuliaTiunova committed:
final fix

JuliaTiunova committed:
add faviacon, fix minor errors

JuliaTiunova committed:
add css

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+    <header class="header">
+      <div class="header__title">
+        <h1 class="title">Bhromaon</h1>
+      </div>
```
Можна просто
```suggestion
      <h1 class="title header__title">Bhromaon</h1>
```

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+      </div>
+      <div class="header__subtitle">
+        <p class="subtitle">Safe Travel Anywhere</p>
+      </div>
```
І тут
```suggestion      
      <p class="subtitle header__subtitle">Safe Travel Anywhere</p>      
```

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+        <form action="#" class="form">
+          <input
+            class="header__search serch-field"
+            type="text"
```
```suggestion
            type="search"
```

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+              <div class="why__wrapper">
+                <div class="why__title">
+                  <h2 class="title-2">Why choose us</h2>
+                </div>
```
```suggestion
                <h2 class="title-2 why__title">Why choose us</h2>
```

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+                    <img
+                      class="why__img why__img--1 img"
+                      src="img/Image_2.png"
+                      alt="beautiful house on the water"
```
👍 👍 👍

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+                    <img
+                      class="why__img why__img--2 img"
+                      src="img/Image_1.png"
+                      alt=" a beautiful scenery of person on a cliff in the mountains"
```
👍 👍

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+              </div>
+            </div>
+          </div>
+          <div class="row align-items-center">
+            <div class="col-8 offset-2">
+              <div class="steps__line">
+                <svg
+                  class="line"
+                  viewbox="0 0 874 92"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    opacity="0.502"
+                    d="M1 91C1 91 217.014 -110.278 442.118 91C442.118 91 639.939 -111.5 873 91"
+                    stroke="white"
+                    stroke-linecap="round"
+                    stroke-dasharray="2 5"
+                  />
+                </svg>
+              </div>
+            </div>
```
Цікаве рішення з розміщенням лінії👍

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+      <section class="what">
+        <div class="what__title">
+          <h2 class="title-2">What Our Tourists Say</h2>
+        </div>
```
Не помилка, але я думаю, що чим менше коду, тим краще=)
```suggestion
        <h2 class="title-2 what__title">What Our Tourists Say</h2>
```

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+                class="what__img--mobile"
+                src="img/ReviewPhoto.png"
+                alt="tourist looking at a mountain scenery"
+              />
```
Тут можна обыйтись лише елементом **img**. Почитай про властивість **object-fit** [MDN](https://developer.mozilla.org/ru/docs/Web/CSS/object-fit)

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+    <footer class="footer" id="contact">
+      <div class="footer__title">
+        <h3 class="title-3">Bhromaon</h3>
+      </div>
```
```suggestion
      <h3 class="title-3 footer__title">Bhromaon</h3>
```

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+        <h3 class="title-3">Bhromaon</h3>
+      </div>
+      <div class="footer__subtitle">
+        <p class="subtitle--small">Safe Travel Anywhere</p>
```
Якщо є модифікатор, то повинен бути і блок
```suggestion
        <p class="subtitle subtitle--small">Safe Travel Anywhere</p>
```

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+        <p class="subtitle--small">Safe Travel Anywhere</p>
+      </div>
+      <div class="footer__social">
+        <ul class="social__list">
```
Є елемент **list** блоку **social**, але немає блоку **social**
```suggestion
      <div class="social footer__social">
        <ul class="social__list">
```

antkl commented on index.html:
```diff
@@ -4,10 +4,312 @@
+                </div>
+                <div class="what__text">
+                  <blockquote class="text">
+                    <sup> <i class="icon-quote-left what__card-quote"></i></sup>
```
Краще зробити блок **quote** і лапки додати через  `::before`
```suggestion
                  <blockquote class="quote text">                    
```

antkl commented on scss/_article.scss:
```diff
@@ -0,0 +1,23 @@
+  }
+
+  &__title {
+    @media (max-width: 991.98px) {
```
Можна зробити міксін, щоб легше було реюзати)

antkl commented on scss/_button.scss:
```diff
@@ -0,0 +1,55 @@
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 20px;
+  list-style-position: 0.07em;
```
тут напевно опечатка
```suggestion
  letter-spacing: 0.07em;
```

antkl commented on scss/_button.scss:
```diff
@@ -0,0 +1,55 @@
+
+  &:hover {
+    background-position-x: -100%;
+    transition: all 0.5s;
```
уже вказано в селекторы вище
```suggestion
```

antkl commented on scss/_button.scss:
```diff
@@ -0,0 +1,55 @@
+  transition: all 0.5s;
+
+  &:hover {
+    background-position-x: -100%;
```
👍

antkl commented on scss/_button.scss:
```diff
@@ -0,0 +1,55 @@
+  &:active {
+    background-position-x: -100%;
+    background-color: lighten($color: $accent, $amount: 10%);
+    transition: all 0.2s;
```
```suggestion
```

antkl commented on scss/_button.scss:
```diff
@@ -0,0 +1,55 @@
+  &--small {
+    min-width: 138px;
+    height: 52px;
+    margin-left: -142px;
```
Якщо реюзати цю кнопку десь ще, це буде заважати. Тому краще перенести цей маржин в` .header__button`.

antkl commented on scss/_header.scss:
```diff
@@ -0,0 +1,69 @@
+  &__search {
+    width: 647px;
+    height: 60px;
+    padding-left: 36px;
```
Це edge case, але краще додати ще й падінг справа, щоб текст не залазив за кнопку
![image](https://user-images.githubusercontent.com/34746432/117638280-56d06300-b18b-11eb-8ad1-ef84e9829028.png)

antkl commented on scss/_link.scss:
```diff
@@ -0,0 +1,22 @@
+  transition: all 0.2s;
+  &:hover {
+    color: $accent-darker;
+    text-shadow: 20px 20px 50px $white, 20px -20px 50px $white,
+      -20px -20px 50px $white, -20px 20px 50px $white;
```
цікавий ефект 👍

antkl commented on scss/_what.scss:
```diff
@@ -0,0 +1,48 @@
+      display: none;
+
+      @media (max-width: 991.98px) {
+        display: inline-block;
+        width: 100%;
```
Картинка і так **inline-block**, а нам треба зробити її **block** щоб не було такого 
![image](https://user-images.githubusercontent.com/34746432/117639167-4bca0280-b18c-11eb-9fb9-e1692a8859f0.png)

```suggestion
        display: block;
```

antkl reviewed (approved):
Дуже хороша робота, класне використання БЕМ, грубих помилок немає, тільки невеличкі рекомендації=) Approve!

JuliaTiunova committed:
fix:
delete extra blocks;
add media mixins;
fix image in 'what' section;
fix input width & header__button margin
